docs/operating-procedures.md

```diff
@@ -50,43 +50,50 @@ The number of `idr-omeroreadonly*` servers may vary, check `/etc/hosts` oni `idr
 ### Bio-Formats cache regeneration
 
 Every time a new version of the software that invalidates the Bio-Formats
-cache files is deployed, it might be necessary to regenerate these files
+cache files is deployed, it will be necessary to regenerate these files
 ahead of time.
 
-Cache files should be regenerated for each imported fileset. The following SQL 
-script will return a list of all the IDs for the first Image in each Fileset:
+Bio-Formats cache files should be regenerated for each imported fileset. For large databases, this operation can be parallelized across the available OMERO
+servers (read-write and read-only). 
 
-    $ echo "copy (select r.i from (select f.id, i.id as i, rank()
-                  over (partition by f.id order by i.id) as x
-                  from fileset f, image i where f.id = i.fileset)
-                  as r where r.x = 1) to stdout;" > first_image_per_fileset.sql
-
-For large databases, cache regeneration should be run in a parallelized manner
-across the available OMERO servers (read-write and read-only):
+Using the OMERO read-write server as the master node, the following commands
+create a list of all remote servers, creates a local SSH key and copies it 
+across nodes to prevent `PermissionsDenied` errors:
 
     $ grep -oE 'omero[^ ]+$' /etc/hosts > nodes
-
-To prevent `PermissionDenied`, create a SSH key and copy it across nodes:
-
     $ ssh-keygen
     $ for i in $(cat nodes); do ssh-copy-id -i ~/.ssh/id_rsa $i; done
 
 Bio-Formats cache regeneration makes use of the
-[OMERO.CLI render plugin](https://pypi.org/project/omero-cli-render/). First
-of all, make sure the plugin is installed on each OMERO server:
+[OMERO.CLI render plugin](https://pypi.org/project/omero-cli-render/). The plugin can be installed on each node as:
 
     $ for i in $(cat nodes); do ssh $i 'pip install --user omero-cli-render'; done
 
 The existing Bio-Formats cache can be moved from the read-write server as:
 
-    $ ssh omeroreadwrite sudo mkdir /data/BioFormatsCache/backup
-    $ ssh omeroreadwrite sudo mv /data/BioFormatsCache/* /data/BioFormatsCache/backup
+    $ sudo mkdir /data/BioFormatsCache/backup
+    $ sudo mv /data/BioFormatsCache/* /data/BioFormatsCache/backup
```

Comment:
Perhaps add something about deleting old backups?

Reply:
Since the `prod70` process ended up being smooth and time-efficient and we can always recover previous Bio-Formats caches from older volumes, I think I would be comfortable replacing these two lines with a more simple:

`$ sudo rm -rf /data/BioFormatsCache/*`

Reply:
I have pushed a commit changing the two lines